Monika After Story/game/script-songs.rpy

```diff
@@ -1191,6 +1191,37 @@ label mas_song_lamour_toujours:
     m 5ekbsa "I want nothing more than to be by your side forever, [player]~"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_songs_database,
+            eventlabel="mas_song_god_knows",
+            category=[store.mas_songs.TYPE_SHORT],
+            prompt="God Knows",
+            random=True,
+            aff_range=(mas_aff.AFFECTIONATE,None)
+        ),
+        code="SNG"
+    )
+
+label mas_song_god_knows:
+    m 1hua "{i}~You know that{w=0.2}{nw}"
+    extend 1eub " I would follow you, no matter what we go through~"
+    m 1efb "{i}~Bring all the darkness the world can offer~"
+    m 1eua "{i}~Because you’ll shine{w=0.2} no matter if the future is bleak~"
+
+    m 3tub "{i}~We’ll aim out{w=0.2} just beyond the boundary~"
+    m 3hka "{i}~And even if it scares me~"
+    m 1hub "{i}~Nothing can shatter my soul because your way is my way~"
+    m 1eub "{i}~Forever on this railway~"
+    m 1fubla "{i}~As if we were God blessed~"
```

Comment:
Do you mean 1fublb, with Resting on hands, Closed (Sad), Line Blush and Open Smile? Couldn't find 1fubla in the EXP PREVIEW anywhere. Is it an unreleased blush type?

Reply:
1fubla is resting on hands, soft eyes, up eyebrows, line blush, and smile. 1dubla is the same except with sad closed eyes. As for the missing sprites, I've seen the devs say multiple times it's better to just leave sprite generation to them. I'm guessing doing it yourself and putting it in the pr creates conflicts on merge.

Reply:
Yeah wasn't sure if I should bother mentioning sprites for that reason, I guess you can just resolve them

Reply:
More that it causes conflicts on this pr if you add sprites and another pr which also adds sprites gets merged.

Then the conflicts here need to be resolved